orchestration/dagster_orchestration/hca_orchestration/config/resources/dataflow_beam_runner/global.yaml

```diff
@@ -0,0 +1,9 @@
+image_name: us.gcr.io/broad-dsp-gcr-public/hca-transformation-pipeline
+image_version: latest
+kubernetes_service_account: monster-dagster
+subnet_name: blarg
```

Comment:
this turned up as thus far undefined during testing. will try to hunt down the expected value.